src/components/NotesPage/NotesPageTopBar.js

```diff
@@ -24,7 +24,6 @@ const anchorPositionMenu = { top: 60, left: 0 };
 
 export const NotesPageTopBar = () => {
   const hasUploads = useSelector(Selectors.hasUploads);
-  // const hasActiveUploads = useSelector(Selectors.hasActiveUploads);
```

Comment:
Please add this commented line back, we will need this code for another task in the future.